guide/miniscript-wallet-simple.md

```diff
@@ -0,0 +1,366 @@
+      alt:
+      caption: The transaction needs to be signed by one additional co-signer.
+    - file: prevent-unlock/path-refresh-cosigner-notification
+      alt:
+      caption: As soon as the refresh transaction has been signed by a co-signer, Bob receives another push notification.
+    - file: prevent-unlock/path-refresh-path-overview-after
+      alt:
+      caption: All funds have been consolidated into one UTXO which expires in 180 days.
+images_path-reset:
+    - file: path-reset/path-reset-push-notification
+      alt:
+      caption: 
+    - file: path-reset/path-reset-app-home
+      alt:
+      caption: The wallet home screen shows a notification that one additional key set is active.
+    - file: path-reset/path-reset-path-overview-before
+      alt:
+      caption: It is clearly indicated which UTXOs are unlocked and which are still locked.
+    - file: path-reset/path-reset-transaction-overview
+      alt:
+      caption: The application tells the user how the activation lock reset works. In the background, UTXOs will be consolidated.
+    - file: path-reset/path-reset-transaction-waiting
+      alt:
+      caption: The transaction has been sent to the other co-signers for approval.
+    - file: path-reset/path-reset-cosigner-notification
+      alt:
+      caption: As soon as the refresh transaction has been signed by a co-signer, Bob receives another push notification.
+    - file: path-reset/path-reseet-path-overview-after
+      alt:
+      caption: All funds have been consolidated into one UTXO which expires in 180 days.
+    - file: path-reset/path-reset-transactions
+      alt:
+      caption: On the wallet home screen, the reset transaction is shown in the transaction history.
+images_device-replacement:
+    - file: device-replacement/replacement-start
+      alt:
+    - file: device-replacement/replacement-wallet-settings
+      alt:
+    - file: device-replacement/replacement-wallet-keys
+      alt:
+    - file: device-replacement/replacement-key-details
+      alt:
+    - file: device-replacement/replacement-how-it-works
+      alt:
+    - file: device-replacement/replacement-pairing-flow
+      alt:
+    - file: device-replacement/replacement-confirmation
+      alt:
+    - file: device-replacement/replacement-keys-update-needed
+      alt:
+---
+
+<!--
+
+Editor's notes
+
+Simple Miniscript wallet reference design.
+
+Illustration sources
+
+https://www.figma.com/community/file/968416729557947210
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/miniscript-wallet-simple/cover.png"
+   retina = "/assets/images/guide/miniscript-wallet-simple/cover@2x.png"
+   mobile = "/assets/images/guide/miniscript-wallet-simple/cover-mobile.png"
+   mobileRetina = "/assets/images/guide/miniscript-wallet-simple/cover-mobile@2x.png"
+   alt-text = "Image showing a wallet with recovery key sets"
+   caption = ""
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Savings wallet with recovery path 
```

Comment:
If this becomes a sub-page of the savings wallet, how about changing the title to "Time-based recovery"?